pkg/gadgets/run/tracer/tracer.go

```diff
@@ -177,6 +177,51 @@ func (t *Tracer) Close() {
 	}
 }
 
+func ringbufAvailable() bool {
+	_, err := ebpf.NewMap(&ebpf.MapSpec{
+		Type:       ebpf.RingBuf,
+		MaxEntries: uint32(os.Getpagesize()),
+	})
+
+	return err == nil
+}
+
+func (t *Tracer) loadeBPFObjects(opts ebpf.CollectionOptions, bufName string) error {
```

Comment:
Keep the param name more specific?

```suggestion
func (t *Tracer) loadeBPFObjects(opts ebpf.CollectionOptions, tracerMapName string) error {
```